### ISSUE TITLE
Question - Does screenrecorderlib include any options for image/video output rotation

### ISSUE THREAD

**Issue by 5cutters** [sskodje/ScreenRecorderLib#336]
I had a quick search and found some references in the code to DXGI_MODE_ROTATION but i think these are internal.

Does screenrecorderlib offer any options to record a screen/window but rotate it in the output recording (90,180,270 degrees)?

**Comment by sskodje** [comment]
No, i'm afraid not. You are correct that there is some internal code to rotate images, but that is because the source textures may be flipped and need correction.

It wouldn't be terribly difficult to implement if you want to have a crack at it, basically a property would need to be added to the source models dictating the orientation, then in the `WriteNextFrameToSharedSurface` method of each recording source, `TextureManager::RotateTexture` would need to be called on the texture.

**Comment by 5cutters** [comment]
Thanks, that might be beyond my skills at the moment but maybe by this time next year!

The question is answered anyway so i'm happy for this to be closed, or you can leave it open as a possible future feature request if you want.
